src/wallet/serviceWorker/request.ts

```diff
@@ -166,7 +166,7 @@ export namespace Request {
             message.type === "SIGN" &&
             "tx" in message &&
             typeof message.tx === "string" &&
-            ("inputIndexes" in message
+            ("inputIndexes" in message && message.inputIndexes !== undefined
```

Comment:
```suggestion
            ("inputIndexes" in message && message.inputIndexes != undefined
```
non-strict equal should also handle case where inputIndexes is "null", am I right @coderabbitai ?

Reply:
@bordalix let's do that no ?